pose_evaluation/metrics/signclip_distance_metric.py

```diff
@@ -0,0 +1,49 @@
+from pose_evaluation.metrics.base_embedding_metric import NumpyArrayEmbeddingMetric
+from typing import Literal
+import numpy as np
+from tqdm import tqdm
+from scipy.spatial.distance import cosine
+import torch
+import torch.nn.functional as F
+
+
+class SignCLIPEmbeddingDistanceMetric(NumpyArrayEmbeddingMetric):
```

Comment:
Perhaps what we want is a `Similarity` metric (higher is better) than a `Distance` metric. There is no "cosine distance" as far as I understand

Reply:
I had to look it up myself. It's a thing, and I think some papers use one and some the other. But cosine similarity does seem to be more common